### ISSUE TITLE
missing autoload.php in manual download 

### ISSUE THREAD

**Issue by mujeebasif** [XeroAPI/xero-php-oauth2#27]
Regarding manual installation readme.md at root of repo says 
> Download the files and include autoload.php:

 i have downloaded files from 
https://github.com/XeroAPI/xero-php-oauth2/releases
and/or by clicking download button at github 
i.e. https://github.com/XeroAPI/xero-php-oauth2/archive/master.zip
in both cases i'm missing mentioned autoload.php 
let me know if missing something. 
Thanks

**Comment by SidneyAllen** [comment]
Hi @mujeebasif 

The Xero PHP SDK uses composer to manage the downloading and initializing of the library.

I recommend watching this short video on how to create a composer file and download the SDK for use.

https://www.youtube.com/watch?v=UYRYlXhdxGU&feature=youtu.be

The autoload.php file is part of Composer "vendor" file

And you'll use this bit of code to load the "xero-php-oauth2" library
require __DIR__ . '/vendor/autoload.php';

Let us know how this works out for you.

**Comment by PaHOUSE** [comment]
So just to confirm, there's no choice other than to download composer which itself requires PHP locally with something like XAMPP, then watch the esoteric video on how to download the SDK using the awful composer UI. And all this is supposed to make life easier?

Would it not be kinder to have a zipped version which includes everything for the rest of us idiots? Why is the zipped version even available if it's essentially incomplete?

Either way can you confirm or deny the aforementioned arsing about is or isn't essential? If not how do we use it without the autoload.php file?

**Comment by calcinai** [comment]
@PaulHous It is possible to use without composer, you'll just need an `include` for each file (and its dependencies, which are not part of this repo) before you can use it.

It is very possible (and free) to make a tar/gzip with all the dependencies as part of there build pipeline using something like Travis each time there's a release on Github.  I can't speak for the maintainers, but that might be a middle ground they're willing to accept.

That said, it really is best practice to use composer–it's been around for many years now and has become the de-facto official dependency manager for php.  If you work as a php developer, I guarantee it'll make your life easier once you get your head around it.  

TL;DR The simplest way for you to get up and running would be to [download composer](https://getcomposer.org/download/) into the root of your local copy of this repository, then simply run `php composer.phar install` and that'll generate you an autoload file and pull the additional dependencies.  From that point everything in the docs should work as described.

Although the above will work, it'll be a strange structure for your project as it'll essentially make `xero-php-oauth2` "your" project.  Hope that was helpful.
